lib/salus/scanners/yarn_audit.rb

```diff
@@ -46,6 +46,7 @@ def run
         # vulns were all whitelisted
         return report_success if vulns.empty?
 
+        add_line_number(vulns)
```

Comment:
Another minor but I'd be a fan of moving the logic you've added here to a stand alone class for working with the line number pieces.  We will have a coupling of passing the vulns hash around but if we document the required hash keys I think it'll be a nice separation of logic here.
```
YarnLock.new('yarn.lock').add_line_number(vulns)
```

Reply:
done